week-7/Homework/mandatory/1-debugging-practice/script.js

```diff
@@ -29,16 +29,22 @@ const check = document.getElementById("check");
 //via Book function and start render function
 function submit() {
   if (
-    title.value == null ||
-    title.value == "" ||
-    pages.value == null ||
-    pages.value == ""
+    title.value === null ||
+    author.value === "" ||// I changed title to author  5
+    pages.value === null ||
+    pages.value === ""
   ) {
     alert("Please fill all fields!");
     return false;
   } else {
-    let book = new Book(title.value, title.value, pages.value, check.checked);
-    library.push(book);
+    let book = new Book(title.value, author.value, pages.value, check.checked); // I changed title to author 6
+    myLibrary.push(book);//Changerd library to myLibrry   3
+    //makes the form empty after submit.
+    title.value = "";
+    author.value= "" ;
+    pages.value= null;
+    check.checked= false;
```

Comment:
Nice! This isn't strictly a bug, but it's good to think about the user's experience.